modules/rosa-autorepair-ocm.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * rosa_cluster_admin/rosa_nodes/rosa-managing-worker-nodes.adoc
+// * nodes/rosa-managing-worker-nodes.adoc
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="rosa-autorepair-ocm_{context}"]
+= Configuring AutoRepair on a machine pool using {cluster-manager}
+
+You can configure machine pool AutoRepair for your {product-title} cluster by using {cluster-manager-first}.
+
+.Prerequisites
+
+* You created a {product-title} (ROSA) cluster.
+* You have an existing machine pool.
+
+.Procedure
+
+
+. Navigate to {cluster-manager-url} and select your cluster.
+. Under the *Machine pools* tab, click the Options menu {kebab} for the machine pool that you want to configure auto repair for.
```

Comment:
This step can be confusing: When you click the kebab icon associated with the machine pool, you will see the Edit and Delete options. To change the AutoRepair setting, you need to select Edit to open the Edit Machine Pool dialog, where you can enable or disable AutoRepair."
Suggested step would be, 

- Under the *Machine pools* tab, click the Options menu {kebab} for the machine pool that you want to configure auto repair for.
- From the context menus, select Edit.
- In the Edit Machine Pool dialog that appears, find the AutoRepair option.
- Select or clear the box next to *AutoRepair* to enable or disable.
- Click *Save* to apply the change to the machine pool.

Reply:
added your suggestion